src/shaders/grass_lod2.geom

```diff
@@ -108,22 +130,17 @@ void main()
 
     vec3 normal = (gs_in[0].Normal + gs_in[1].Normal + gs_in[2].Normal) / 3;
 
-    highp float noise_u, noise_v;
-
     bool tall = true;
     for (int i=0; i<=N_GRASS_STRAWS; i++) {
         frag_pos = gs_in[0].FragPos + frag_pos_01 * COORDS_U[i] + frag_pos_02 * COORDS_V[i];
-        tex_coord = tex_coord_base_01 * COORDS_U[i] + tex_coord_base_02 * COORDS_V[i];
-
-        noise_u = 2 * rand((i+1) * gs_in[i % 3].worldPos.xy) - 1;
-        noise_v = 2 * rand((i+1) * gs_in[i % 3].worldPos.zx) - 1;
+        tex_coord = gs_in[0].TexCoord + tex_coord_base_01 * COORDS_U[i] + tex_coord_base_02 * COORDS_V[i];
 
         if (tall)
```

Comment:
This is old, already merged code, but this branch is really big. Could affect performance. Probably make an issue on it as a possible improvement.